### PR TITLE
3112/duplicates visual qa

### PR DESCRIPTION
**Pull request by ColinBuyck**
# Pull Request Template

## Issue Overview

This PR addresses #3112 

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This PR includes various bug fixes with the exception of **number 6 and 9 in the issue**. Per conversations with @slowbot, 6 has been removed from this ticket's requirements and 9 is being further investigated but will be a simple fix once the min-height value is finalized.

## How Can This Be Tested/Reviewed?

This can be tested by signing into the partners portal and clicking through to the duplicates feature. For those bug fixes related to the AG table or header, they should be present on all table views (not just the duplicates)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: ui-components/src/headers/PageHeader.scss [hunk 1]
```diff
@@ -42,6 +42,7 @@
 
 .page-header__breadcrumbs {
   margin-bottom: 10px;
+  margin-top: calc(-1 * var(--bloom-s4));
```
Comment (ColinBuyck) [inline ui-components/src/headers/PageHeader.scss:45]: Can't negate css variable directly so had to use this workaround

File: ui-components/src/tables/AgTable.tsx [hunk 2]
```diff
@@ -207,8 +207,12 @@ const AgTable = ({
 
   return (
     <div className={`ag-theme-alpine ag-theme-bloom ${className || ""}`}>
-      <div className="flex justify-between flex-col md:flex-row">
-        <div className={`flex flex-wrap ${showSearch ? "mb-5" : "hidden"}`}>
+      <div
+        className={`flex justify-between flex-col md:flex-row ${
+          (showSearch || headerContent) && "my-4"
```
Comment (ludtkemorgan) [inline ui-components/src/tables/AgTable.tsx:212]: This is outputting `undefined` when headerContent is undefined and showSearch is false. It's working, but we should make sure nothing is outputted in that case
image

**Review by ludtkemorgan — APPROVED** [review]
The functionality all looks good. Just one minor comment

File: ui-components/src/tables/AgTable.tsx (hunk 2 above)
Comment (ColinBuyck) [reply ui-components/src/tables/AgTable.tsx]: Fixed 🎾

**Review by emilyjablonski — APPROVED** [review]
Screen Shot 2022-10-31 at 1 47 19 PM

LGTM! One question, for # 2 I think I would have interpreted this as the filter button still being in line with the left nav, just with reducing top margin for both. I'm good either way though!

**Comment by ColinBuyck** [comment]
@emilyjablonski Touched base with Jesse and he agreed with your interpretation. Thank you for the catch and I've updated the PR to look like the screenshot below!
Screen Shot 2022-10-31 at 12 02 57 PM

**Pull request merged**